lib/streamlit/elements/widgets/options_selector/feedback_utils.py

```diff
@@ -0,0 +1,125 @@
+# Copyright (c) Streamlit Inc. (2018-2022) Snowflake Inc. (2022-2024)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import annotations
+
+from typing import Callable, Literal
+
+from streamlit.elements.widgets.options_selector.options_selector_utils import (
+    MultiSelectSerde,
+)
+from streamlit.proto.ButtonGroup_pb2 import ButtonGroup as ButtonGroupProto
+
+_thumb_icons = [":material/thumb_up:", ":material/thumb_down:"]
```

Comment:
Aren't all of these module-level constants? In that case it need upper-case and an added `Final` type instruction to indicate that these are not changed at runtime.